### PR TITLE
Add callables

### PR DESCRIPTION
**Pull request by lferraz**
#### Changes

Fixes # (issue)

#### Type of change

- [ ] 📚  Documentation Update
- [ ] 🧪 Tests Cases
- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔬 New feature (non-breaking change which adds functionality)
- [ ] 🚨 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 📝 This change requires a documentation update

#### Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Did you update CHANGELOG in case of a major change?

### CODE REVIEW

File: limbus/core/param.py [hunk 1]
```diff
@@ -484,6 +487,18 @@ def disconnect(self, dst: "Param" | IterableParam) -> None:
         self._disconnect(self, dst)
 
 
+class PropParam(Param):
```
Comment (edgarriba) [inline limbus/core/param.py]:
```suggestion
class PropertyParam(Param):
```

File: limbus/core/params.py [hunk 2]
```diff
@@ -1,40 +1,26 @@
...
-        setattr(self, name, Param(name, tp, value, arg, self._parent))
-
-    def __getattr__(self, name: str) -> Param:  # type: ignore  # it should return a Param
-        """Trick to avoid mypy issues with dinamyc attributes."""
-        ...
 
+    @abstractmethod
+    def declare(self, *args, **kwargs) -> None:
+        """Add or modify a param."""
+        raise NotImplementedError
```
Comment (edgarriba) [inline limbus/core/params.py:22]:
```suggestion
        raise NotImplementedError

```

**Review by edgarriba — APPROVED** [review]
LGTM. missing small tests

File: limbus/core/params.py (hunk 2 above)
Comment (lferraz) [reply limbus/core/params.py:22]: I do not understand teh suggested change... it is the same code as it is XD

File: limbus/core/param.py (hunk 1 above)
Comment (lferraz) [reply limbus/core/param.py]: ok, I used Prop because was shorter, I will change it

File: limbus/core/params.py (hunk 2 above)
Comment (edgarriba) [reply limbus/core/params.py:22]: end of line was empty i believe

**Pull request merged**